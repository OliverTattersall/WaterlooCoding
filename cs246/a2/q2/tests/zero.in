create a +infinity
create b 0
create c 0
create d 20
+ b c
+ a b
- b c 
- d b 
/ b d
/ d b
* b c 
* b d
u- b
u- d
print a
