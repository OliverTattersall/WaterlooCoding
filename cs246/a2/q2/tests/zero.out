0
+infinity
0
20
0
+infinity
0
0
0
-20
+infinity
